site/gatsby-site/blog/using-ai-to-connect-ai-incidents/index.mdx

```diff
@@ -46,7 +46,7 @@ You can find the first integration of this tool available at [the report submiss
 * Iz Beltagy, Matthew E. Peters, Arman Cohan at [AllenAI](https://allenai.org/) for the Longformer model as [hosted on Huggingface](https://huggingface.co/docs/transformers/model_doc/longformer)
 * Luna McNulty and Sean McGregor for ongoing front-end work interfacing with the API
 
-[^1]: Having completed their capstone, the authors will all be starting Software Engineering internships this July, with Nicholas Broce at **PCC Structurals Inc**, Nicholas Olson at **Cognex**, and Jason Scott-Hakanson at **Lam Research**.
+[^1]: Having completed their capstone, the authors will all be starting Software Engineering work this July, with Nicholas Broce at **Garmin AT**, Nicholas Olson at **Cognex**, and Jason Scott-Hakanson at **Lam Research**.
```

Comment:
Nick said he was starting in August, and saying "this July" is awkward with this landing right at the end of July. I think we could change it to "this summer."

Reply:
Thanks, updated!